src/FNA3D_Driver_Vulkan.c

```diff
@@ -109,6 +109,8 @@ static const uint32_t deviceExtensionCount = SDL_arraysize(deviceExtensionNames)
 #define NULL_SAMPLER (VkSampler) 0
 #define NULL_RENDER_PASS (VkRenderPass) 0
 
+#define DEFAULT_PIPELINE_CACHE_FILE_NAME "vk_pipeline.cache"
```

Comment:
This should probably be more descriptive, like FNA3D_Vulkan_PipelineCache.blob, so developers know where the new file came from.